IdentityCore/src/network/MSIDHttpRequest.m

```diff
@@ -127,7 +127,7 @@ - (void)sendWithBlock:(MSIDHttpRequestDidCompleteBlock)completionBlock
               
               MSID_LOG_WITH_CTX(MSIDLogLevelVerbose,self.context, @"Parsed response: %@, error %@, error domain: %@, error code: %ld", _PII_NULLIFY(responseObject), _PII_NULLIFY(error), error.domain, (long)error.code);
               
-              if (responseObject && _shouldCacheResponse)
+              if (responseObject && self->_shouldCacheResponse)
```

Comment:
Would you give a try to remove the underscore in front of ```_shouldCacheResponse```

Reply:
good catch.